src/BasicAbility/Masa.Contrib.BasicAbility.Auth/Service/UserService.cs

```diff
@@ -93,6 +93,13 @@ public async Task<UserModel> GetCurrentUserAsync()
         return await _callerProvider.GetAsync<object, UserModel>(requestUri, new { id }) ?? new();
     }
 
+    public async Task<StaffDetailModel?> GetCurrentStaffAsync()
+    {
+        var userId = _userContext.GetUserId<Guid>();
+        var requestUri = $"api/staff/getExternalByUserId";
+        return await _callerProvider.GetAsync<object, StaffDetailModel>(requestUri, new { userId });
+    }
+
```

Comment:
157 line. Possible null reference argument for parameter 'json' in 'T? JsonSerializer.Deserialize<T>(string json, JsonSerializerOptions? options = null)'.